### DISCUSSION TITLE
Sending Outlook emails

### DISCUSSION THREAD

**Discussion by anouar-rogue** [symfony/symfony#46672] [Q&A]
I did some research but i always find solutions that use mailgun or other providers like sendgrid, even the official doc doesn't contain much detail on this matter.
![mail](https://user-images.githubusercontent.com/106612172/173665697-2b61f1eb-5dbf-44d0-a0a1-e07c982c4b12.png)

If someone just can share with me a link for an appropriate kinda of solution. I appreciate the help

**Comment by avtarsm** [comment]
The default symfony built-in transport works fine for office 365 based email.

The mailer dns should be like

MAILER_DSN=smtp://emailaddress:password@smtp.office365.com:587?encryption=STARTTLS

Change the emailaddress and password to the actual email address you want to use along with the password. If it is office 365 based account, please make sure that the email address is allowed to send email via SMTP. This can be done via Microsoft Exchange setting for the email address.

**Reply by zippytiff** [reply]
Hi

Would i be right in thinking this will now stop working, as Office 365 moves to Modern Authentication in the next few months ?

Thx

ZT

**Comment by dbu** [comment]
i had to figure this out too :cry: 

finally got it working, wrote up my solution at https://gist.github.com/dbu/3094d7569aebfc94788b164bd7e59acc
